server/src/main/java/io/druid/guice/http/JettyHttpClientModule.java

```diff
@@ -116,6 +117,9 @@ public HttpClient get()
 
       httpClient.setIdleTimeout(config.getReadTimeout().getMillis());
       httpClient.setMaxConnectionsPerDestination(config.getNumConnections());
+      final QueuedThreadPool queuedThreadPool = new QueuedThreadPool(config.getNumMaxThreads());
+      queuedThreadPool.setName(QueuedThreadPool.class.getSimpleName() + "@" + queuedThreadPool.hashCode());
```

Comment:
Can this be more descriptive? something like `JettyHttpClientModulePool@HASH` or similar?